src/nevermined/utils/WebServiceConnector.ts

```diff
@@ -192,14 +206,48 @@ export class WebServiceConnector extends Instantiable {
         )
     }
 
+    public async fetchCID(cid: string): Promise<string> {
+        
+        const url = this.config.ipfsGateway + '/api/v0/cat?arg=' + cid.replace('cid://', '')
+        const authToken = WebServiceConnector.getIPFSAuthToken()
+        const options = {
+            method: 'POST',
+            ...(authToken && {
+            headers: { Authorization: `Basic ${authToken}` }
+            })
+        }
+
+        return fetch(url, options)
```

Comment:
better use try catch, easier to read

Reply:
try/catch is not necessary here because the catch is just throwing the exception. Modifying the code to remove unnecessary stuff